Packs/CommunityCommonDashboards/ReleaseNotes/2_1_0.md

```diff
@@ -0,0 +1,6 @@
+
+#### Dashboards
+
+##### New: CISOMetrics_copy
```

Comment:
Hi @rurhrlaub, 
Please provide an explanation about the dashboard, for example:
- What's its purpose?
- What does it show and why?
etc.

Reply:
DONE